lib/active_shipping/carriers/fedex.rb

```diff
@@ -460,6 +460,7 @@ def build_location_node(xml, name, location)
           xml.StreetLines(location.address1) if location.address1
           xml.StreetLines(location.address2) if location.address2
           xml.City(location.city) if location.city
+          xml.StateOrProvinceCode(location.state)
```

Comment:
I tested this locally and the freight request to the FedEx server now succeeds. It was returning an error that State was missing. With the edit to the gem, I no longer see the error, but instead see rate quotes for my FedEx freight account.